tests/system-defaults-write.nix

```diff
@@ -10,6 +10,7 @@
   system.defaults.NSGlobalDomain.ApplePressAndHoldEnabled = true;
   system.defaults.NSGlobalDomain.AppleShowAllExtensions = true;
   system.defaults.NSGlobalDomain.AppleShowScrollBars = "Always";
+  system.defaults.NSGlobalDomain.AppleScrollerPagingBehavior = true;
```

Comment:
It's probably enough to only include options with more special values in the test, there's not really anything that can go wrong with a boolean option I think. 😄